src/Offer/Commands/UpdatePriceInfo.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\Commands;
+
+use CultuurNet\UDB3\PriceInfo\PriceInfo;
+
+final class UpdatePriceInfo extends AbstractCommand
+{
+    protected PriceInfo $priceInfo;
```

Comment:
I wonder if this could not just be using `CultuurNet\UDB3\Model\ValueObject\Price\PriceInfo`
That can also be used inside the new command handler where it gets converted to the old format before sending it to the aggregate.

Reply:
Having started on this, I think this would be best handled in another PR as to not make PR's too big.